components/ui/Text.tsx

```diff
@@ -16,7 +17,7 @@ export const Text: React.FC<TextProps> = (props: TextProps) => {
   ];
 
   return (
-    <RNText style={textStyles} numberOfLines={props.numLines}>
+    <RNText style={textStyles} {...testID(props.testID)}>
```

Comment:
numberOfLines prop got removed. Please correct it

Reply:
resolved